examples/bri-3/src/bri/communication/agents/bpiMessagesStorage.agent.ts

```diff
@@ -32,7 +31,7 @@ export class BpiMessageStorageAgent extends PrismaService {
     });
   }
 
-  async createNewBpiMessage(bpiMessage: BpiMessage): Promise<BpiMessage> {
+  async storeNewBpiMessage(bpiMessage: BpiMessage): Promise<BpiMessage> {
```

Comment:
Seeing  a lot of overlap with the last PR, as well as some conflicts, do you resolving?

Reply:
Rebased and pushed. Now you should only see new commits.